### PR TITLE
Implemented Trigger in Oscilloscope UI

### PR DESCRIPTION
**Pull request by akarshan96**
Partly Fixes issue #260 

Changes: 
- Replaced Trigger TextView to Checkbox.
- Implemented Trigger in Oscilloscope UI.
- Resolved some typo errors

[apk link](https://drive.google.com/file/d/0B423XEmWL9vKbDdSamJlUElTMUE/view?usp=sharing) (W1 is producing sine wave at the frequency of 3000)

Screenshots for the change: 
screenshot_1499195124

### CODE REVIEW

**Comment by CloudyPadmal** [comment]
Hello @akarshan96 ! Shall we add the label `WIP` to this since it partially fixes the issue? :)

**Comment by akarshan96** [comment]
@CloudyPadmal, I have divided issue #260 into 4 sub-issues/tasks. Since this PR deals with the completion of a single task done, it's ready to merge. I would have added WIP if it was incomplete. :)

File: app/src/main/res/layout/fragment_timebase_tigger.xml [hunk 1]
```diff
@@ -67,36 +70,44 @@
...
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:id="@+id/tv_tigger_label_tt"
+                android:id="@+id/checkbox_trigger_tt"
                 android:layout_alignStart="@+id/tv_timebase_label_tt"
                 android:layout_below="@+id/seekBar2"
                 tools:layout_editor_absoluteX="318dp"
                 app:layout_constraintTop_toBottomOf="@+id/seekBar_timebase_tt"
-                android:layout_gravity="center_vertical"
+                android:layout_gravity="center_vertical|center_horizontal"
```
Comment (viveksb007) [inline app/src/main/res/layout/fragment_timebase_tigger.xml]: Was only ```center``` giving some different results?

**Comment by viveksb007** [comment]
@akarshan96 Just tested the apk. Great work :+1:

**Pull request merged**